src/parser/core/modules/BuffWindow.tsx

```diff
@@ -55,13 +55,18 @@ interface BuffWindowRequiredGCDs {
 	severityTiers: SeverityTiers
 }
 
-interface BuffWindowTrackedCooldown {
-	action: Action
-	expectedPerWindow: number,
+interface BuffWindowTrackedActions {
+	actions: BuffWindowTrackedAction[]
+	iconAction: Action
```

Comment:
Given it's only being used for the icon, makes more sense to just `icon: string` this, and let consumers pick if they want to source that from an action or not.